src/resources/test_data_sample.json

```diff
@@ -0,0 +1,17 @@
+[
+    {
+        "attributes": {
+            "aggregation_project": "RP-1545",
+            "location": "OnPrem",
+            "md5": "gs://fc-d40787e1-06e7-40e7-af19-134b48dd88a1/RP-1545/Exome/16177_CCPM_1300040_BLOOD/v4/16177_CCPM_1300040_BLOOD.bam.md5",
+            "hard_clipping_used": False,
```

Comment:
If these are going to stay in the repo - this should be `false` to be valid JSON